### DISCUSSION TITLE
Potential scrolling bug?

### DISCUSSION THREAD

**Discussion by aeharding** [inokawa/virtua#406] [General]
Hi there! I don't have enough information to make a bug report, but there may be an issue with newer Virtua versions causing scrolling to flick around in rare cases.

Unfortunately it doesn't happen that often so I don't have a reproduction or video recording. But I thought I'd make a post in case anyone else has any ideas, and to put it on your radar. :)

I think it may have happened upgrading Virtua 0.26.0 -> 0.27.5, but I'll let you know with more information when I get it.

(BTW, I think it also happens on iOS - but I've only had it happen very very rarely.)

https://lemmy.world/post/12870811

**Comment by aeharding** [comment]
Update: It is an odd issue. Apparently this item got rendered twice:

I'm not sure why this would happen, and it is still very rare, so I am continuing investigating. Hopefully get a video soon!

https://lemmy.world/comment/8251690

<img src="https://github.com/inokawa/virtua/assets/2166114/7aabfda9-ddce-4ee1-b1bb-7d788356834b" width="250">

**Reply by aeharding** [reply]
Another update - I got the video from the user! Also they confirmed that it was not happening in older versions of Virtua (0.26 was good I believe)


https://github.com/inokawa/virtua/assets/2166114/f1187a45-963f-4fc5-b71e-87c52d7a4675

**Reply by inokawa** [reply]
Thank you for the video. Looks very strange...
For now I have no idea why this happens (maybe related to overscan?) but we can fix it if there is reproduction.

**Reply by aeharding** [reply]
Of course, I will let you know if/when I find out more! Thanks for taking a look!

**Reply by meotimdihia** [reply]
@inokawa I am getting this problem too. Our users are reporting this:

<img src='https://github.com/inokawa/virtua/assets/300961/50a2868e-326f-49c1-8d6b-cc542d716dc2' width="200" />
<img src='https://github.com/inokawa/virtua/assets/300961/9f86f75a-92fa-4c47-8e90-f372dd1d16d3' width="200" />

These bugs are happening with Xiaomi or Realme phones. Though I just asked users but can't reproduce it.

**Reply by aeharding** [reply]
Another user report - https://github.com/aeharding/voyager/issues/1520

**Reply by inokawa** [reply]
Thank you. Personally, I've been focusing on improving stability of virtua lately. It would be helpful if there were reprodutable situation.
